test/puppetlabs/puppetdb/testutils/services.clj

```diff
@@ -110,11 +110,10 @@
 (defmacro with-puppetdb-instance
   "Convenience macro to launch a puppetdb instance"
   [& body]
-  `(with-redefs [sutils/db-metadata (delay {:database "HSQL Database Engine"
-                                            :version [2 2]})]
+  `(with-redefs [sutils/db-metadata (delay (sutils/db-metadata-fn))]
```

Comment:
I just mimicked what we do elsewhere for fetching the db-metadata, since were not always gonna have an HSQLDB connection in this function anymore we have to get the metadata dynamically.